src/PerformanceTests/Tests/Categories/Base.cs

```diff
@@ -39,10 +39,14 @@ public virtual void GatedPublishRunner(Permutation permutation)
 
         void Tasks(Permutation permutation, [CallerMemberName] string memberName = "")
         {
-            permutation.Category = GetType().Name;
+            var fixture = GetType().GetCustomAttribute<TestFixtureAttribute>();
+            permutation.Category = fixture.Category;
+            permutation.Description = fixture.Description;
```

Comment:
We could use the test name instead, but the point was to use it as a filter if we need to filter out the result.